CONTRIBUTING.md

```diff
@@ -0,0 +1,98 @@
+# Contributing
+
+:tada: Thank you for being interested in contributing to the SNARK artifacts project! :tada:
+
+Feel welcome and read the following sections in order to know how to ask questions and how to work on something.
+
+All members of our community are expected to follow our [Code of Conduct](/CODE_OF_CONDUCT.md). Please make sure you are welcoming and friendly in all of our spaces.
+
+We're really glad you're reading this, because we need volunteer developers to help this project come to fruition. 👏
+
+## Issues
+
+The best way to contribute to our projects is by opening a [new issue](https://github.com/privacy-scaling-explorations/snark-artifacts/issues/new/choose) or tackling one of the issues listed [here](https://github.com/privacy-scaling-explorations/snark-artifacts/contribute).
+
+## Pull Requests
+
+Pull requests are great if you want to add a feature or fix a bug. Here's a quick guide:
+
+1. Fork the repo.
+
+2. Run the tests. We only take pull requests with passing tests.
+
+3. Add a test for your change. Only refactoring and documentation changes require no new tests.
+
+4. Make sure to check out the [Style Guide](/CONTRIBUTING.md#style-guide) and ensure that your code complies with the rules.
+
+5. Make the test pass.
+
+6. Commit your changes.
+
+7. Push to your fork and submit a pull request on our `main` branch. Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us.
```

Comment:
```suggestion
1. Fork the repo.

<!-- 2. Run the tests. We only take pull requests with passing tests. -->

<!-- 3. Add a test for your change. Only refactoring and documentation changes require no new tests. -->

<!-- 4. Make sure to check out the [Style Guide](/CONTRIBUTING.md#style-guide) and ensure that your code complies with the rules. -->

<!-- 5. Make the test pass. -->

2. Commit your changes.

3. Push to your fork and submit a pull request on our `main` branch. Please provide us with some explanation of why you made the changes you made. For new features make sure to explain a standard use case to us.
```

We don't have any tests (yet?). So don't you think we should not mention tests?

Reply:
Right!